transfering-a-cookbook.md

```diff
@@ -0,0 +1,38 @@
+# Transferring a Cookbook to the Chef Brigade
+
+Got a cookbook you'd like help with? We'd love to help!
+
+## Before you begin
+
+We need to work with the github repo owner and supermarket cookbook owner. If you aren't this person, let us know and we'll try to contact them. If necessary, fine beverages and chocolate chip cookies may be provided to encourage them to work with us.
+
+You'll need:
+
+- Code Repository URL
+- Supermarket cookbook name
+
+## Transferring the code
+
+We're assuming the cookbook code is already in its own repo in GitHub. If it's in a shared repo, or hosted elsewhere, it'll require a slightly different approach.
+
+
+- From the repository home page in GitHub
+- Go to the **Settings** page
+- Scroll down to the **DangerZone** and click **Transfer**
+- Enter the appropriate **repo name** and for the **New owner's GitHub username or organization name** enter `chef-brigade`
+- Click **I understand, transfer this repository.**
+
+
+## Transferring the cookbook in Supermarket
+
+- From the cookbook home page in Supermarket
+- Click **Manage Cookbook** and select **Transfer Ownership**
+- Enter `chef-brigade` and click **Transfer**
+
+
+## Cleanup links to the old home
+
+There are probably many references to the old URLs out there in the world. Some places to check
+
+- Update the `README.md` with a link to the current repo and supermarket page
+- Update the **Source URL** and **Issues URL** in supermarket
```

Comment:
mention also that these are changeable in the `metadata.rb` of the cookbook (`issues_url` and `source_url`)